spec/rubocop/cop/ezcater/feature_flag_active_spec.rb

```diff
@@ -0,0 +1,39 @@
+# encoding utf-8
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Ezcater::FeatureFlagActive, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:msgs) { [described_class::MSG] }
+
+  it "accepts a call with a tracking_id" do
+    source = %[EzFF.active?("FlagName", tracking_id: "user:23456")]
+    inspect_source(source)
+    expect(cop.offenses).to be_empty
+  end
+
+  it "accepts a call with an identifiers array" do
+    source = %[EzFF.active?("FlagName", identifiers: ["user:23456", "user:12345"])]
+    inspect_source(source)
+    expect(cop.offenses).to be_empty
+  end
+
+  it "errors when neither expected key is present" do
+    source = %[EzFF.active?("FlagName", bad_key: "user:123", other_bad_key: ["user:123", "user:456"])]
+    inspect_source(source)
+    expect(cop.messages).to match_array(msgs)
+  end
+
+  it "errors when only one argument is passed" do
+    source = %[EzFF.active?("FlagName")]
+    inspect_source(source)
+    expect(cop.messages).to match_array(msgs)
+  end
+
+  it "works with the full module name" do
```

Comment:
This is not a change that absolutely needs to happen but it's good practice to explore a broader space of inputs:

Thoughts on using a bit of metaprogramming to ensure that we test each case for both `EzFF` and `EzcaterFeatureFlag`? You could also add in a bit of randomization to ensure that your test cases aren't tied to a particular implementation:

```
let(:flag_name) { generate(:feature_flag_name) }
let(:tracking_id) do
  # some code to generate a random id-like value
end
let(:message) { "..." }

before { inspect_source(line) }

%w(EzFF EzcaterFeatureFlag).each do |constant_name|
  describe "a line referring to the #{constant_name} constant" do
    context "and correctly uses a tracking_id" do
      let(:line) { "#{constant_name}.active?(#{flag_name}, tracking_id: #{tracking_id})" }
      
      it "does not report an issue" do
        expect(cop.messages).not_to include(msg)
      end
    end
    
    # ...
  end
end
```